test-links.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Test Wiki Links
+---
+
+# Test Wiki Links
+
+This is a test page to verify wiki link resolution:
+
+- Link to [[1a]] should resolve to /zk/1a/
+- Link to [[Sascha2020]] should resolve to /zk/Register/Sascha2020/
+- Link to [[0b1a]] should resolve to /zk/0b1a/
+- Link to [[nonexistent]] should fallback to /nonexistent/
```

Comment:
Also test escaping, e.g. wiki links in code span and code block should not. be converted to links.